index.html

```diff
@@ -1,17 +1,80 @@
 <!doctype html>
-<form name="path" action="path.html">
-  <p><label>cache base<br><input name="cache_base" value="https://tmpnix-gsdrv.glitch.me/drvs/"></label></p>
-  <p><label>hash<br><input name="hash" value="52xwnqlqa6l4bbf01yifahb517zax89d"></label></p>
-  <p><input type="submit" value="view path"></p>
-</form>
-<form name="drv" action="drv.html">
-  <p><label>cache base<br><input name="cache_base" value="https://tmpnix-gsdrv.glitch.me/drvs/"></label></p>
-  <p><label>hash<br><input name="hash" value="52xwnqlqa6l4bbf01yifahb517zax89d"></label></p>
-  <p><label>cache base bin<br><input name="cache_base_bin" value="https://cdn.glitch.me/35f4f809-f949-484d-af9c-269b3b7abc78/"></label></p>
-  <p><input type="submit" value="view drv"></p>
-</form>
-<form name="nar" action="nar.html">
-  <p><label>cache base<br><input name="cache_base" value="https://cdn.glitch.me/35f4f809-f949-484d-af9c-269b3b7abc78/"></label></p>
-  <p><label>hash<br><input name="hash" value="s3jcnk777vd7vjr1agqn6zc0bpaz07sr"></label></p>
-  <p><input type="submit" value="view nar"></p>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width,initial-scale=1">
+<title>Nix cache view</title>
+<style>
+  body {
+    margin: 1.5rem;
+    font-family: sans-serif;
+  }
+  .wide {
+    box-sizing: border-box;
+    width: 100%;
+    max-width: 30rem;
+  }
+</style>
+<p id="presets_container" hidden>
+  <label>Presets</label><br>
+  <select id="presets">
+    <option></option>
+    <option value="https://cache.nixos.org/" selected data-sample-hash="7ghhnlwla2mddkg7hgqa5v0sr8g5hga8">cache.nixos.org</option>
+    <option value="https://tmpnix-gsdrv.glitch.me/drvs/" data-cache-base-out="https://cdn.glitch.me/35f4f809-f949-484d-af9c-269b3b7abc78/" data-sample-hash="52xwnqlqa6l4bbf01yifahb517zax89d">wh0/tmpnix-drv</option>
+    <option value="https://cdn.glitch.me/35f4f809-f949-484d-af9c-269b3b7abc78/" data-cache-base-drv="https://tmpnix-gsdrv.glitch.me/drvs/" data-sample-hash="s3jcnk777vd7vjr1agqn6zc0bpaz07sr">wh0/tmpnix-out</option>
+  </select>
+</p>
+<form name="omni">
+  <p><label>Cache base<br><input name="cache_base" value="https://cache.nixos.org/" class="wide"></label></p>
+  <p><label>Cache base drv<br><input name="cache_base_drv" value="" class="wide"></label></p>
+  <p><label>Cache base out<br><input name="cache_base_out" value="" class="wide"></label></p>
```

Comment:
changing cache_base_bin to cache_base_out. need to update drv.html correspondingly